packages/node/src/lib/sdk.ts

```diff
@@ -0,0 +1,57 @@
+import { createAndBind } from '@sentry/core';
+import { NodeOptions } from './backend';
+import { NodeFrontend } from './frontend';
+
+export { addBreadcrumb, captureEvent, setUserContext } from '@sentry/core';
```

Comment:
Same here. Duplicated core import, unless specific order required (would be nice to mention that if that's the case)